Resources/Prototypes/Roles/Antags/revolutionary.yml

```
@@ -4,6 +4,12 @@
   antagonist: true
   setPreference: true
   objective: roles-antag-rev-head-objective
+  requirements:
+  - !type:OverallPlaytimeRequirement
+  time: 54000 # 15h
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)